config/config.go

```diff
@@ -432,19 +433,28 @@ const (
 )
 
 type UplinkStatus struct {
-	SwIfIndex    uint32
-	TapSwIfIndex uint32
-	LinkIndex    int
-	Name         string
-	IsMain       bool
-	Mtu          int
+	SwIfIndex           uint32
+	TapSwIfIndex        uint32
+	LinkIndex           int
+	Name                string
+	IsMain              bool
+	Mtu                 int
+	PhysicalNetworkName string
+
+	FakeNextHopIP4 net.IP
+	FakeNextHopIP6 net.IP
+}
+
+type PhysicalNetwork struct {
+	VrfId     uint32
+	PodVrfId  uint32
+	SwIfIndex uint32
```

Comment:
I don't think we should be storing `SwIfIndex` as part of the PhysicalNetwork object as there might be multiple Uplinks in a given PhysicalNetwork.